pubspec.yaml

```diff
@@ -37,6 +37,7 @@ dependencies:
   intl: ^0.16.1
   sprintf: ^5.0.0
   flutter_svg: ^0.19.1
+  sip_ua: ^0.3.1
```

Comment:
Please add the library notice to the `THIRD_PARTY_NOTICES.md` file

Reply:
Niemals!